api/src/pcapi/use_cases/save_venue_bank_informations.py

```diff
@@ -30,8 +36,12 @@ def __init__(
         self.venue_repository = venue_repository
         self.bank_informations_repository = bank_informations_repository
 
-    def execute(self, application_id: str):
-        application_details = get_venue_bank_information_application_details_by_application_id(application_id)
+    def execute(self, application_id: str, procedure_id: Optional[str] = None):
+        if not procedure_id:
+            procedure_id = settings.DMS_VENUE_PROCEDURE_ID
```

Comment:
Si je comprends bien, l'argument est optionnel temporairement, n'est-ce pas ? Une fois le déploiement effectué, on aura forcément une valeur, si j'ai bien compris. Est-ce qu'on se rajoute un FIXME pour supprimer ce `if not procedure_id` et supprimer le `Optional` ? Idem pour le `Optional` de la fonction `bank_information_job()` ?

Reply:
Je n'avais pas pris conscience qu'effectivement, je pourrais faire le ménage dès à présent.
Les cas avec le `procedure_id` vide (dans mon code) sont les tests et la création de coordonnées bancaires pour une structure, procédure qui sera décommissionnée une fois les dossiers traités.
Mais +1 pour supprimer l'optionnel dès maintenant